src/frontend/mame/infoxml.cpp

```diff
@@ -434,7 +434,7 @@ void info_xml_creator::output(std::ostream &out, const std::function<bool(const
 		prepared_info() = default;
 		prepared_info(const prepared_info &) = delete;
 		prepared_info(prepared_info &&) = default;
-		prepared_info &operator=(const prepared_info &) = delete;
+		prepared_info &operator=(const prepared_info &) = default;
```

Comment:
Why have you changed this?  The object is written this way to prevent accidental copy assignment, which is expensive.  The objects shouldn’t be copied at all, only moved.

Reply:
So is it instantiating the copy assignment operator when a `std::queue` is instantiated there?  I don’t see any code where it tries assigning an lvalue reference to a queue element – it only emplaces default-constructed elements, and sets elements with rvalues returned by the task lambda.

The only copy I can see is at line 517, but that’s a copy of an rvalue anyway.  The local `pi` is constructed with the result of `std::future::get` which returns by value.

Is this a regression in a new version of the Microsoft C++ standard library?  If it is, it needs to be reported.